MdeModulePkg/Include/Guid/MemoryBucketInformation.h

```diff
@@ -0,0 +1,35 @@
+/** @file
+Guid for Pei storing and retreiving the PEI memory buckets.
+**/
+
+#ifndef _MEMORY_BUCKET_INFORMATION_H_
+#define _MEMORY_BUCKET_INFORMATION_H_
+
+#define MEMORY_BUCKET_INFORMATION_GUID \
+{ 0x36138737, 0xb6db, 0x4ed6, { 0x9b, 0x4b, 0xf9, 0x61, 0x28, 0xe7, 0x19, 0x3c } }
+
+extern EFI_GUID  gMemoryBucketInformationGuid;
+
+#define EFI_MEMORY_TYPE_STATISTICS_GUID \

```

Comment:
Is this from the EFI spec?

Reply:
If it is something locally invented just drop the `EFI_` prefix.

Reply:
Changed to not include EFI.